### PR TITLE
[11559-xCluster][Doc] Documentation on how to setup xCluster in K8s when you have pod to pod connectivity

### PR DESCRIPTION
**Pull request by Arjun-yb**
Adding section to the xCluster setup docs for how to setup xCluster in K8s when you have pod to pod connectivity.

### CODE REVIEW

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 1]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
       000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
     ```
 
+## xCluster Setup in K8s ( pod to pod connectivity )
+
+1. Create 2 Universes (source and target)
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:185]: Done

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 2]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
...
+2. Create tables in both source and target universes
+  (At source)
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:187]: Added

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 3]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
...
+  ```.sh
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:188]: Changed

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 4]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
...
+
+
+  (At target)
+  ```.sh
+  kubectl exec -it -n <target_universe_namespace> -t <target_universe_master_leader> -c <target_universe_container> -- bash
+  /home/yugabyte/bin/ysqlsh -h <target_universe_yqlserver>
+  create table query
+  ```
+
+  For Example
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:209]: Done

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 5]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
...
+  ```.sh
+  kubectl exec -it -n xcluster-source -t yb-master-2 -c yb-master -- bash -c \
+  "/home/yugabyte/bin/yb-admin -master_addresses yb-master-2.yb-masters.xcluster-target.svc.cluster.local, \
+  yb-master-1.yb-masters.xcluster-target.svc.cluster.local,yb-master-0.yb-masters.xcluster-target.svc.cluster.local \
+  setup_universe_replication ac39666d-c183-45d3-945a-475452deac9f_xCluster_1 \
+  yb-master-2.yb-masters.xcluster-source.svc.cluster.local,yb-master-1.yb-masters.xcluster-source.svc.cluster.local, \
+  yb-master-0.yb-masters.xcluster-source.svc.cluster.local 00004000000030008000000000004001"
+  ```
+
+5. Do some DML at source side and observe the replication at target side
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:238]: Updated

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 6]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
...
+
+  For Example
+  ```.sh
+  kubectl exec -it -n xcluster-target -t yb-master-2 -c yb-master -- bash
+  /home/yugabyte/bin/ysqlsh -h yb-tserver-1.yb-tservers.xcluster-target
+  create table employees(id int primary key, name text);
+  ```
+
+3. Collect table UUIDs, go to the `Tables` section in the Admin UI (127.0.0.1:7000)
+4. Setup Replication from source universe
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:217]: Done

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 7]
```diff
@@ -180,6 +180,93 @@ When both universes use different certificates, you need to store the certificat
...
+  ```
+
+  For Example
+  ```.sh
+  kubectl exec -it -n xcluster-target -t yb-master-2 -c yb-master -- bash
+  /home/yugabyte/bin/ysqlsh -h yb-tserver-1.yb-tservers.xcluster-target
+  create table employees(id int primary key, name text);
+  ```
+
+3. Collect table UUIDs, go to the `Tables` section in the Admin UI (127.0.0.1:7000)
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:216]: Done

**Review by ajcaldera1 — CHANGES_REQUESTED** [review]
So we probably should put a link to how it is done via platform as well: https://docs.yugabyte.com/latest/yugabyte-platform/create-deployments/async-replication-platform/ 
This can be done at the bottom of the page. Something like: For users of Yugabyte Platform (2.11+), please visit the following link to configure replication. 

The other thing we need to talk about is how networking needs to be setup. So the instructions from multi-gke cluster setup need to be followed with some modifications to make sure that `kube-dns` is properly configured with stub domains to enable cross cluster DNS.  The python script on that page needs a small modification to handle setting up multiple clusters simultaneously. Working on perfecting/writing that up.

**Comment by Arjun-yb** [comment]
Thanks @ajcaldera1 
 Added the instruction for how to setup xCluster via Platform.
 Once I get the instructions for multi-gke cluster setup, I can work on documenting that.

**Comment by bmatican** [comment]
@ajcaldera1 Let's decouple the multi-cluster docs from this, so we can make incremental progress! If this is good enough to be useful, for k8s setup, please stamp this as is and we can file a new issue for follow up.

**Review by ajcaldera1 — APPROVED** [review]
LGTM

**Comment by bmatican** [comment]
Thanks Alan!

**Pull request merged**